### PR TITLE
Updated DESCRIPTION and structure of application so that it is more l…

### PR DESCRIPTION
**Pull request by ck2136**
I've essentially migrated your code into a `golem()` shiny app packaging framework. There are a few things you could add (i.e. `unit tests`). A issue was the name of the app (i.e. `r_regex_tester_app`) was not an appropriate package name so I made up something quick. If you think it make sense maybe creating a new repository and then migrating the code there might be good. I don't expect you to take all these changes either! Try 'em out and see if the package behaves the same way as it was before.

### CODE REVIEW

**Comment by AdamSpannbauer** [comment]
I'll review and let you know about any edits.

RE your comment about tests: I likely won't be adding any tests (although it is, of course, good practice) or doing much else in the repo at all.  This is an old project that I don't have much motivation to maintain at this point.  So any tests/other updates will be up to contributors like yourself.

File: DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Imports: 
+    shiny,
+    golem,
+    data.table,
+    shinyBS,
+    stringr,
+    purrr,
+    tidyr,
+    htmltools,
+    processx
```
Comment (AdamSpannbauer) [inline DESCRIPTION]: Some initial feedback.  Given that #10 appears to be the main motivation for `golem`-ifying the app, I think we should be exhaustive about the dependencies of the app here.  The below is one of the warnings from running `devtools::check()`.

> ```
>❯ checking dependencies in R code ... WARNING
>  '::' or ':::' imports not declared from:
>    ‘DT’ ‘RColorBrewer’ ‘rvest’ ‘shinythemes’ ‘xml2’
> ```

There are other issues brought up by running this check, but as of now, I'm not planning on deploying the package to CRAN.  If you'd like to get the package to a CRAN-ready state, then we would need to get all Errors, Warnings, & Notes down to 0.

Comment (ck2136) [reply DESCRIPTION]: I certainly haven't thought about getting the app CRAN-ready but maybe I could push that forward. I've added/updated the dependencies.

**Comment by AdamSpannbauer** [comment]
Hi @ck2136, sorry for the delay in getting back to you on this PR.  I should have some time this week to review everything.  Thanks for all the time/effort you put into this.

**Comment by AdamSpannbauer** [comment]
Hi, @ck2136.  I'm sorry for failing on following up on this, and thanks for the effort you put into it.  I ended up taking your advice on using `golem`.  However, I took this as an excuse to learn `golem` and I ended up creating and merging #16.

**Pull request closed without merging**